### PR TITLE
Remove deprecated datasources config property

### PR DESCRIPTION
**Pull request by Praveen2112**
Datasources property in ServerConfig is depricated and has not been in use for quite
a long time.

### CODE REVIEW

**Comment by Praveen2112** [comment]
@electrum . Can you please review them ?

File: presto-main/src/main/java/io/prestosql/server/PrestoServer.java [hunk 1]
```diff
@@ -183,9 +183,7 @@ private static void updateConnectorIds(Announcer announcer, CatalogManager metad
         // build announcement with updated sources
```
Comment (kokosing) [inline presto-main/src/main/java/io/prestosql/server/PrestoServer.java]: Please fix commit message:

```
Remove deprecated datasources config property
```

File: presto-main/src/main/java/io/prestosql/server/PrestoServer.java [hunk 2]
```diff
@@ -156,10 +154,7 @@ private static void updateConnectorIds(Announcer announcer, CatalogManager metad
...
         ServiceAnnouncement announcement = getPrestoAnnouncement(announcer.getServiceAnnouncements());
 
-        // get existing connectorIds
-        String property = nullToEmpty(announcement.getProperties().get("connectorIds"));
-        List<String> values = Splitter.on(',').trimResults().omitEmptyStrings().splitToList(property);
-        Set<String> connectorIds = new LinkedHashSet<>(values);
+        Set<String> connectorIds = new LinkedHashSet<>();
 
         // automatically build connectorIds if not configured
         if (connectorIds.isEmpty()) {
```
Comment (kokosing) [inline presto-main/src/main/java/io/prestosql/server/PrestoServer.java]: `connectorIds.isEmpty()` is always `true`. Please try to remove `connectorIds` variable and optimize the if block.

File: presto-main/src/main/java/io/prestosql/server/PrestoServer.java [hunk 3]
```diff
@@ -183,9 +178,7 @@ private static void updateConnectorIds(Announcer announcer, CatalogManager metad
         // build announcement with updated sources
         ServiceAnnouncementBuilder builder = serviceAnnouncement(announcement.getType());
         for (Map.Entry<String, String> entry : announcement.getProperties().entrySet()) {
```
Comment (electrum) [inline presto-main/src/main/java/io/prestosql/server/PrestoServer.java]: We can now replace this loop with
```java
builder.addProperties(announcement.getProperties());

**Comment by Praveen2112** [comment]
@electrum , @kokosing I have resolved those changes.

**Pull request merged**